docs/src/components/ChunkedSnippet.astro

```diff
@@ -1,16 +1,27 @@
 ---
+import  fs  from "node:fs/promises"
+import path from "node:path"
+import { execSync } from 'node:child_process';
 import prettier from 'prettier';
 import { Code } from 'astro:components';
 import { CodeBlock } from '@interledger/docs-design-system'
-const {source, chunk} = Astro.props;
+const {source, chunk, l} = Astro.props;
 import * as phpPlugin from '@prettier/plugin-php'
 
-// Retrieve code snippet from GitHub as text
-const getApiData = async () => {
-  const data = await fetch(source).then((response) => response.text());
-  return data;
-};
-const code = await getApiData();
+const gitRoot = execSync('git rev-parse --show-toplevel').toString().trim();
+const snippetsPath = path.join(gitRoot, 'snippets');
+
+const getChunk = async () => {
+  if (source.startsWith('http')) {
+    const data = await fetch(source).then((response) => response.text());
+    return data;
+  }
```

Comment:
For now, still allowing the `http` path to be fetched from the `source`. This is to support the PHP snippets, which will be done in a separate PR